src/EPR.RegulatorService.Facade.Core/Services/Producer/ProducerService.cs

```diff
@@ -37,7 +43,12 @@ public async Task<HttpResponseMessage> GetOrganisationDetails(Guid userId, Guid
 
         _logger.LogInformation("Attempting to fetch organisation details for organisation'{externalId}'", externalId);
 
-        return await _httpClient.GetAsync(url);
+        var uriBuilder = new UriBuilder(_httpClient.BaseAddress)
```

Comment:
Please undo this change, the uriBuilder.Path returns encoded Url and if there is ? in the Url it will be converted to 3f something, ignore the SQ warning as peter is happy to leave it as is